src/Pages/saemsolyoo/Main/Feeds/Feed/Comments/Comment/Comment.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import "./Comment.scss";
+
+class Comment extends Component {
+  render() {
+    return (
+      <div className="Comment">
+        {this.props.commentList.map((comment) => {
+          return (
+            <li key={comment.id}>
+              <div>
+                <a href="#">{comment.userName}</a>
+                <span>{comment.content}</span>
+              </div>
+              <div>
+                <i
+                  id={comment.id}
+                  className="fas fa-trash"
+                  onClick={this.props.deleteComment}
+                ></i>
```

Comment:
```suggestion
                <i
                  id={comment.id}
                  className="fas fa-trash"
                  onClick={this.props.deleteComment}
                />
```
- 셀프 클로징 태그를 써줄 수 있겠네요!
- 특별히 id 셀렉터를 써주신 이유가 있을까요?

Reply:
부모 컴포넌트에서 filter 함수를 이용해서 댓글을 삭제하려고 버튼에 id를 줬었습니다. 그런데 몇일전에 준식님께서 다른분께 설명해주신 방법을 전해듣고 다시 그부분 반영해서 수정해볼게요!!

Reply:
① 셀프클로징 수정 완료했습니다.
② 자식컴포넌트에서 arrow function을 이용해 부모컴포넌트 함수에게 인자를 전달하는 방법으로 댓글삭제 기능 구현을 수정했습니다.
  --> 처음에  
  <i   className="fas fa-trash"
         onClick={this.props.deleteComment(comment.id)}  />  이렇게 해도 되는 줄 알고 했다가 계속 전달값이 콘솔에 찍히지 않아서 결국은 이벤트가 발생하는 버튼에 id를 주고 e.target.id를 이용해서 삭제필터링을 해줬었는데, 
         onClick={() => this.props.deleteComment(comment.id)}    이렇게 해보니 이제 삭제가 잘 됩니다!!